suites/pacific/rados/tier-3_rados_mondb_trimming.yaml

```diff
@@ -0,0 +1,173 @@
+# Suite contains tests related to mon db trimmings
```

Comment:
Can we rename this file to `tier-3_rados_test-mon-db-trimming.yaml`?

Reply:
renamed the suite file